app/controllers/houseConsignment/index/documents/AddAnotherDocumentController.scala

```diff
@@ -67,9 +67,10 @@ class AddAnotherDocumentController @Inject() (
             case true =>
               Redirect(
                 controllers.houseConsignment.index.documents.routes.TypeController
-                  .onPageLoad(arrivalId, mode, houseConsignmentIndex, viewModel.nextIndex)
+                  .onPageLoad(arrivalId, mode, NormalMode, houseConsignmentIndex, viewModel.nextIndex)
               )
             case false =>
+              // TODO: pattern match on mode (houseConsignment mode), to decide to go back to cross check page, or next page in add HC journey.
```

Comment:
Does this need to be addressed now or is it for a particular ticket? Might be worth popping a ticket number on just encase, if you can.

Reply:
Thanks @Reece-Carruthers 

https://jira.tools.tax.service.gov.uk/browse/CTCP-5046

This will be brought in. There's 3 TODOs it'll have to address.